### DISCUSSION TITLE
download md content from a shared document

### DISCUSSION THREAD

**Discussion by glungtung** [outline/outline#6052] [General]
Hi,
Is there a way to get markdown content of a shared/published document, through a RSS feed or a specific url ?
For example, a URL like app.outline.com/s/xxx-xxx/download

**Comment by tommoor** [comment]
Not currently, the closest would be the `documents.export` endpoint but it doesn't accept a shareId at the moment – authentication is needed

**Reply by glungtung** [reply]
Ok thanks, I think "documents.info" endpoint with a shareId is my best solution for now, where i can find "text" field wich seems to be markdown text also
